packages/expo-dev-menu/ios/DevMenuManager.swift

```diff
@@ -2,6 +2,18 @@
 
 import EXDevMenuInterface
 
+public class DefaultDevMenuDelegate : DevMenuDelegateProtocol {
```

Comment:
What would you say about `DevMenuBridgeProxyDelegate`? Does it have to be `public`?
```suggestion
public class DevMenuBridgeProxyDelegate: DevMenuDelegateProtocol {
```